src/Tribe/Tickets.php

```diff
@@ -1832,6 +1769,71 @@ public static function tickets_own_stock( $post_id ) {
 			return true;
 		}
 
+		/**
```

Comment:
If we've removed all calls to this function, why aren't we removing it?
The file it registers is also removed.

Reply:
If you would have an older version of ET+ running it could cause a fatal.  We could completely remove and then in the dependency manager update the required version. This way seems a bit safer.

Reply:
Gotcha, makes sense!